packages/@o3r/eslint-config/package.json

```
@@ -0,0 +1,125 @@
+{
+  "name": "@o3r/eslint-config",
+  "version": "0.0.0-placeholder",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Recommended eslint v9+ configuration for Otter project",
+  "exports": {
+    "./package.json": {
+      "default": "./package.json"
+    },
+    ".": {
+      "default": "./src/index.js"
```

I think you want to target the dist/ folder
```suggestion
      "default": "./dist/index.js"
```
(same for the other below)